compiler/src/iree/compiler/Dialect/Util/Analysis/Constant/ConstExpr.cpp

```diff
@@ -161,9 +160,8 @@ ConstExprAnalysis::ConstExprAnalysis(Operation *rootOp) {
         for (auto &use : definingOp->getUses()) {
           Operation *useOp = use.getOwner();
           // Skip expanding of ops within dispatch or nested regions.
-          if (definingOp->getParentOp() != useOp->getParentOp()) {
+          if (definingOp->getParentOp() != useOp->getParentOp())
             continue;
-          }
```

Comment:
I was told the style in iree is to always have these braces

Reply:
someone started removing them for some reason
runtime still has them (as not having them in C is dangerous with macros)

Reply:
I don't know anymore :P I hate omitting them as it leads to a lot of churn (want to add a debug line? now you need to add braces, etc) - I'd prefer if we kept them, but people working upstream tend not to include them and then we end up with a mix

IMO if something is multi-line then always add them - I've seen insane stuff like for-if-for-if-for-if with multi-line expressions without a single {} and it's insane - single-line in compiler code is ok to omit if it helps readability

Reply:
ok well I also have been trying to add them every time and have been asking for others to also, so I'll keep doing that then. Was just asking because you removed them here, but this was more of a sane case.